px-data-grid-paginated.html

```diff
@@ -574,6 +584,18 @@
           }));
         }
 
+        // Reemit event from px-data-grid
```

Comment:
@alump, why we need reemission in a paginated grid?
Added stopPropagation in order to prevent multiple callings of handlers.

Reply:
Is it needed also to add stopPropagation to https://github.com/vaadin/px-data-grid/blob/7e5236297afc88ed5caae10aea6908ca5c2cf453/px-data-grid-paginated.html#L568
?

Reply:
to allow on-filter-saving="function" you need to do this. maybe addEventListener might work, but this needs to work with attribute mapping too.

Reply:
Ok. my bad, I will remove need to re-emit for those two signals